CONTRIBUTING.md

```diff
@@ -44,9 +44,26 @@ Other notes:
 
 ## Pull Requests
 
-To accommodate review process we suggest that PRs are categorically broken up. 
+To accommodate review process we suggest that PRs are categorically broken up.
 Ideally each PR addresses only a single issue. Additionally, as much as possible
-code refactoring and cleanup should be submitted as a separate PRs from bugfixes/feature-additions. 
+code refactoring and cleanup should be submitted as a separate PRs from bugfixes/feature-additions.
+
+### Process for Reviewing PRs
+
+All PRs require 2 Reviews before merge. When reviewing PRs please use the following annotations:
+
+- `LGTM` without an explicit approval means that the changes look good, but you haven't pulled down the code, run tests locally and thoroughly reviewed it.
+- `Approval` through the GH UI means that you understand the code, documentation/spec is updated in the right places, you have pulled down and tested the code locally.
```

Comment:
Hmmm I do think we should have separate but similar terminology/acronym for when the code is "LGTM"-able, but has not been tested locally (the equivalent of utACK). I don't want to promote laziness here, but just think we should have some verbiage for that 😄

Reply:
I think `LGTM` is the lazy path (i.e. `utACK`) right? Is it not written like that?